test/model/message_list_test.dart

```diff
@@ -0,0 +1,241 @@
+import 'package:checks/checks.dart';
+import 'package:test/scaffolding.dart';
+import 'package:zulip/api/model/events.dart';
+import 'package:zulip/api/model/model.dart';
+import 'package:zulip/api/route/messages.dart';
+import 'package:zulip/model/message_list.dart';
+import 'package:zulip/model/narrow.dart';
+import 'package:zulip/model/store.dart';
+import '../api/fake_api.dart';
```

Comment:
nit: blank line between `package:` and relative imports (like we do in other files)